src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -111,6 +112,14 @@ class IntelliJPlugin implements Plugin<Project> {
         def ivyFile = createIvyRepo(project, extension)
         def version = extension.version
 
+        project.configurations.create("provided")
```

Comment:
`project.configurations.create("provided")` returns create configuration which is can be used later, e.g. while adding dependencies or expanding project classpath